sections/other_rules.tex

```diff
@@ -46,14 +46,21 @@ \subsection*{\hypertarget{Difficulty}{Difficulty}}
 
 All \textbf{Artifacts} received from a starting bonus should be placed into your \textbf{hand} and not shuffled into your Starting Deck.
 If you searched for any Artifacts, shuffle the Artifact Deck and its Discard Pile\index{Discard Pile} together afterwards, and then discard one Artifact from the top to form the Artifact Discard Pile again.\par
-The chosen difficulty also determines the number and type of neutral enemies that are encountered during Neutral Combat according to the table below.
+The chosen difficulty also determines the number and type of neutral enemies that are encountered during Neutral Combat according to the \hyperlink{Difficulty Table}{table} at the back cover of the book.
+
+\end{multicols}
 
 \vfill
-\hfill\includegraphics[width=0.8\linewidth]{\art/cerberi.png}
+\includegraphics[width=\linewidth]{\art/sharpshooter.jpg}
 
-\end{multicols}
+\clearpage
 
 \begin{figure*}[!hb]
   \centering
-  \includegraphics[width=\linewidth]{\tables/field_difficulty.png}
+  \includegraphics[width=\linewidth]{\tables/opt_rules.png}
```

Comment:
```suggestion
  \includegraphics[width=\linewidth]{\tables/optional_rules.png}
```
Make it explicit.

Reply:
BTW, the image is not properly cropped. Can you get rid of the areas beyond the yellow frame?